### ISSUE TITLE
connect/disconnect accounts dinamically

### ISSUE THREAD

**Issue by meskio** [katzenpost/mailproxy#21]
If I understand the API corretcly, right now if I want to connect or disconnect an account I need to `Shutdown` the proxy and `New` another one. Isn't it?

It will be nice to be able to dinamically connect and disconnect accounts.

**Comment by Yawning** [comment]
That's correct.  There's no reason why it's a hard limitation, and the intention has always been to support configuring most things by either API calls or the management interface, but doing so has been a low priority up till now.
